<?xml version="1.0" ?>
<launch>

<!-- Launch the interbotix gazebo robot-->
  <!-- Define arguments for the included launch file -->
  <arg name="show_lidar" default="true" />
  <arg name="use_trajectory_controllers" default="true" />
  <arg name="robot_model" default="locobot_wx250s" />

  <!-- Include the contents of another launch file -->
  <include file="$(find interbotix_xslocobot_gazebo)/launch/xslocobot_gazebo.launch">
    <arg name="show_lidar" value="$(arg show_lidar)" />
    <arg name="use_trajectory_controllers" value="$(arg use_trajectory_controllers)" />
    <arg name="robot_model" value="$(arg robot_model)" />
    <arg name="paused" value="false"/>
  </include>


  <!-- run the saved rviz environment -->
  <node type="rviz" name="rviz" pkg="rviz" args="-d $(find me326_locobot_example)/rviz/rviz_example_env.rviz" />

  <node type="locobot_control.py" name="locobot_control" pkg="me326_locobot_example" respawn="true" output="screen"/>

  <!-- <node pkg="tf2_ros" type="static_transform_publisher" name="odom_broadcaster" args="0 0 0 0 0 0 locobot/odom locobot/base_link" /> -->

</launch>

